web/src/lib/utils.ts

```diff
@@ -269,12 +269,14 @@ export const oauth = {
   isAutoLaunchDisabled: (location: Location) => {
     const values = ['autoLaunch=0', 'password=1', 'password=true'];
     for (const value of values) {
-      if (location.search.includes(value)) {
-        return true;
-      }
+      if (location.search.includes(value)) return true;
     }
     return false;
```

Comment:
Please either revert this change entirely or change it to `return location.search.includes(...)` instead.